config/routes.js

```diff
@@ -20,7 +22,10 @@ export default (router, passport, app) => {
   // Setting up the users api
   router.post('/users', users.create);
   router.post('/users/avatars', users.avatars);
+  api.get('/verify', middleware.verifyToken);
 
+  // Setting up the game api
+  api.post('/game/:id/start', middleware.auth, game);
```

Comment:
Same issue as my 2nd comment directly above this

Reply:
Done